man/systemd.network.xml

```diff
@@ -637,6 +637,13 @@
         </para></listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>IPv6PrefixDelegation</varname></term>
+          <listitem><para>Whether to enable Router Advertisement sending on a link. Defaults
+          to unset. See the <literal>IPv6PrefixDelegation</literal> and <literal>IPv6Prefix</literal>
```

Comment:
please use <varname> for config file settings, not <literal>. Also, suffix with "=", as above.
Hmm, the reference to `IPv6PrefixDelegation=` is kinda recursive, as this section is about precisely this setting, so this doesn't look right.

Reply:
I think I'm missing [ and ] around the section names, I'm trying to point to those two sections, actually. So it should be  <literal>[IPv6PrefixDelegation]</literal>?

Reply:
yeah, if it's a section, use <literal>, if it's an configuration option use <varname> + trailing "="